app/views/projects/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h3> <%= @project.name %> </h3>
+<p> <%= "Material: #{@project.material}" %> </p>
+<p> <%= "Challenge Theme: #{@project.challenge.theme}" %> </p>
+<p> <%= "Number of Contestants: #{@project.contestants.count}" %> </p>
```

Comment:
Even though it's a simple calculation, `.count` is a calculation so per strict MVC conventions, we'd want to see this encapsulated in a model method in the same way you've done with avg_exp. This is a minor MVC violation but is best practice given the guidelines around keeping data logic out of the view